src/test/codegen/unwind-extern-imports.rs

```diff
@@ -0,0 +1,41 @@
+// compile-flags: -C no-prepopulate-passes
+
+#![crate_type = "lib"]
+#![feature(unwind_attributes)]
+
+extern {
+// CHECK: Function Attrs: nounwind
```

Comment:
btw... I'm curious: are these testing directives documented anywhere if I would like learn about codegen tests?

Reply:
https://www.llvm.org/docs/CommandGuide/FileCheck.html has the documentation on FileCheck.

Reply:
There's a [readme in the codegen folder](https://github.com/rust-lang/rust/blob/master/src/test/codegen/README.md) with that link. You just have to look for the docs then you'll find them. ;)